src/auth_api/config.py

```diff
@@ -102,6 +104,8 @@
 TERMS_ACCEPT_URL = \
     f'{SERVICE_URL}{TERMS_ACCEPT_PATH}'
 
+# Invalidate pending login URL
+INVALIDATE_PENDING_LOGIN_URL = '/invalidate'
```

Comment:
We would usually call this `INVALIDATE_PENDING_LOGIN_PATH` as the `_URL` suffix is reserved for full URLs